lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -663,6 +672,31 @@ states:
             auditContext:
               mitigationType: enhanced-verification
 
+  # DWP KBV journey (J7)
+  CRI_DWP_KBV_J7:
+    response:
+      type: cri
+      criId: dwpKbv
+    parent: CRI_STATE
+    events:
+      next:
+        targetState: EVALUATE_GPG45_SCORES
+      access-denied:
+        targetState: PRE_EXPERIAN_KBV_TRANSITION_PAGE
+      enhanced-verification:
+        targetState: MITIGATION_02_OPTIONS_WITH_F2F
+        auditEvents:
+          - IPV_MITIGATION_START
+        auditContext:
+          mitigationType: enhanced-verification
+        checkIfDisabled:
+          f2f:
+            targetState: MITIGATION_02_OPTIONS
+            auditEvents:
+              - IPV_MITIGATION_START
+            auditContext:
+              mitigationType: enhanced-verification
+
   # No photo id journey (M2B)
```

Comment:
I think we also need to add DWP KBV here? see `ADDRESS_AND_FRAUD_M2B`

Reply:
and mitigation journeys - see `MITIGATION_PP_ADDRESS_AND_FRAUD` and `MITIGATION_CHECK_FRAUD_SCORE`